src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -243,15 +245,24 @@ public void send(
       return;
     }
 
-    final Component component =
-        new PersonalizedText(
-            Components.format(
-                format,
-                sender == null
-                    ? new PersonalizedText("Console", ChatColor.AQUA, ChatColor.ITALIC)
-                    : sender.getStyledName(NameStyle.FANCY),
-                new PersonalizedText(message.trim())));
-    match.getPlayers().stream().filter(filter).forEach(player -> player.sendMessage(component));
-    Audience.get(Bukkit.getConsoleSender()).sendMessage(component);
+    if (!sender.isMuted()) {
+      final Component component =
+          new PersonalizedText(
+              Components.format(
+                  format,
+                  sender == null
+                      ? new PersonalizedText("Console", ChatColor.AQUA, ChatColor.ITALIC)
```

Comment:
You can use this: https://github.com/Electroid/PGM/blob/6bc2e673cbb1d180adaa4b4e25d32d6f2717fe6c/src/main/i18n/templates/strings.properties#L33